lib/membrane_live/event.ex

```diff
@@ -253,4 +279,36 @@ defmodule MembraneLive.Event do
     ]
 
   defp event_span_id(id), do: "event:#{id}"
+
+  defp add_playlist_idl_to_peer(peer_hls, playlist_idl, peer_id) do
+    if peer_hls.peer_id == peer_id,
+      do: Map.put(peer_hls, :playlist_idl, playlist_idl),
+      else: peer_hls
+  end
```

Comment:
IMO it's pretty strange that function `add_playlist_idl_to_peer` doesn't always add playlist_idl to peer.